src/JuMP.jl

```diff
@@ -316,6 +316,30 @@ function _try_get_solver_name(model_like)
     end
 end
 
+function _try_set_silent(model_like)
+    try
+        return MOI.set(model_like, MOI.Silent(), true)
+    catch ex
+        if isa(ex, ArgumentError)
+            return "Silent() attribute not implemented by the optimizer."
```

Comment:
What is the `ArgumentError` thrown ?

Reply:
Also tried following the same logic as `solver_name` in the auxiliary `_try_get_solver_name` function but this also does not make much sense, as the argument Silent() should always work independently of the optimizer